pkg/idhubmain/persister.go

```diff
@@ -29,3 +30,20 @@ func initSignedClaimPersister(db *gorm.DB) *claimsstore.SignedClaimPGPersister {
 	db.AutoMigrate(claimsstore.SignedClaimPostgres{})
 	return persister
 }
+
+func initRootClaimPersister(db *gorm.DB) *claimsstore.RootCommitsPGPersister {
+	persister := claimsstore.NewRootCommitsPGPersister(db)
+	db.AutoMigrate(
+		claimsstore.RootCommit{},
+	)
+	return persister
+}
+
+func initTreeStore(db *gorm.DB) db.Storage {
```

Comment:
So there is also `initNodePersister` and `initTreePersister` does the same thing as this func. The only difference is that this returns `db.Storage` interface vs the implemented version. We should do some renaming and consolidate if it makes sense.